x-pack/plugins/asset_manager/server/lib/validators/validate_es_date.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import datemath from '@kbn/datemath';
+import { AssetsValidationError } from './validation_error';
+
+export function validateESDate(dateString: string) {
+  try {
+    datemath.parse(dateString);
+  } catch (error: any) {
+    throw new AssetsValidationError(`"${dateString}" is not a valid Elasticsearch date value`);
```

Comment:
does the error thrown by `datemath` provide useful context that we can pass along ?